pubsub/system-test/quickstart.test.js

```diff
@@ -35,14 +37,18 @@ describe(`pubsub:quickstart`, () => {
     const expectedTopicName = `my-new-topic`;
 
     pubsubMock = {
-      createTopic: (_topicName) => {
+      createTopic: (_topicName, _callback) => {
         assert.equal(_topicName, expectedTopicName);
+        assert.equal(typeof _callback, 'function');
```

Comment:
:+1: We should add callback-checking to the standard, IMO.

@jmdobry Maybe create an issue and/or submit a mini-PR for it, assuming you agree? (Alternatively, I'm happy to do this myself if you'd prefer.)